plugin/src/main/java/io/snyk/eclipse/plugin/html/BaseHtmlProvider.java

```diff
@@ -102,6 +102,7 @@ public String replaceCssVariables(String html) {
 		htmlStyled = htmlStyled.replace("<style nonce=\"ideNonce\" data-ide-style></style>", css);
 		htmlStyled = htmlStyled.replace("var(--default-font)",
 				" ui-sans-serif, \"SF Pro Text\", \"Segoe UI\", \"Ubuntu\", Tahoma, Geneva, Verdana, sans-serif;");
+		htmlStyled = htmlStyled.replace("var(--main-font-size)", "10px");
```

Comment:
Why?

Reply:
Also: can this be taken from the IDE font configuration in Eclipse? See: https://www.perplexity.ai/search/how-to-access-the-eclipse-plat-kwR4S6_ESB29RHq7YGW9kw